rclcpp/src/rclcpp/init_options.cpp

```diff
@@ -44,6 +44,7 @@ InitOptions::InitOptions(const InitOptions & other)
 : InitOptions(*other.get_rcl_init_options())
 {
   shutdown_on_sigint = other.shutdown_on_sigint;
+  initialize_logging_ = other.initialize_logging_;
```

Comment:
I thought I commented this before, but I think I didn't.
I would like to see this fix in a separate PR, so it can be backported.